applications/FluidDynamicsApplication/tests/test_mass_conservation_check_process.py

```diff
@@ -0,0 +1,229 @@
+﻿from __future__ import print_function, absolute_import, division
+
+import KratosMultiphysics
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.kratos_utilities as kratos_utils
+import KratosMultiphysics.FluidDynamicsApplication as KratosFluid
+from KratosMultiphysics.FluidDynamicsApplication.apply_mass_conservation_check_process import ApplyMassConservationCheckProcess
+import sys
+import re
+
+try:
+    import KratosMultiphysics.mpi as KratosMPI
+    import KratosMultiphysics.MetisApplication as KratosMetis           
+    import KratosMultiphysics.TrilinosApplication as KratosTrilinos
+    import KratosMultiphysics.mpi.distributed_import_model_part_utility as distributed_import_model_part_utility
+except:
+    print("Running tests without MPI")
```

Comment:
please improve those imports
Only import what you actually use (I am pretty sure you don't directly use metis)

Also use [this](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/MappingApplication/tests/mapper_test_case.py#L49) for checking in a test if you are running MPI